app/views/welcome/index.html.slim

```diff
@@ -48,17 +48,27 @@
             .row.form-row
               .col-xs-6
                 = f.text_field :first_name, placeholder: "First Name", class: "form-control"
+              .col-xs-6.form-error.first-name-error
```

Comment:
@kienan col-xs-12 knocked the last name field down more than i would like. Pushing a better solution soon (moving the errors to their own row)